server/src/http/feature_refresher.rs

```diff
@@ -392,23 +392,15 @@ impl FeatureRefresher {
                                 }
                             }
                             FeatureError::NotFound => {
-                                warn!("Had a bad URL when trying to fetch features. Removing ourselves");
+                                info!("Had a bad URL when trying to fetch features. Removing ourselves from the list of refresh tasks");
```

Comment:
What is "ourselves" pointing to here? It's a little unclear to me. Judging from the next line, it's the token? Maybe it's clear to the users, but could it be something like 

> Removing the refresh task for this token

or

> Removing this token from the list of refresh tasks

?

Reply:
yeah, agreed.